boards/cats_rev1Pro/src/main.h

```diff
@@ -22,6 +22,8 @@
 /* Includes ------------------------------------------------------------------*/
 #include "stm32l4xx_hal.h"
 
+#define BOOTLOADER_MAGIC_PATTERN 0xAA88BB77
```

Comment:
Whats the Magic pattern? Is that the start of the memory for the bootloader?

Reply:
Will this pattern change with the version?